inc/Database/Replace.php

```diff
@@ -428,4 +428,22 @@ public function set_dry_run( $state ) {
 
 		return $state;
 	}
+
+	/**
+	 * Unserializes data only if it was serialized.
+	 * unserialize function is called with allowed_classes set to false
+	 * to ensure no object instantiation and magic function calls.
+	 *
+	 * @since 3.2.3
+	 *
+	 * @param string $data Data that might be unserialized.
+	 * @return mixed Unserialized data can be any type.
+	 */
+	public function maybe_unserialize_safe( $data ) {
+		if ( is_serialized( $data ) ) {
+			return @unserialize( trim( $data ), array('allowed_classes' => false ) );
```

Comment:
When you do it like that, and plugins save data with classes in options, the data will be corrupted after.